Settings.c

```diff
@@ -603,11 +603,13 @@ static void writeMeterModes(const Settings* this, FILE* fd, char separator, unsi
 int Settings_write(const Settings* this, bool onCrash) {
    FILE* fd;
    char separator;
+   char *filename;
```

Comment:
Please initialize to `NULL` to avoid undefined/uninitialized values `filename` in case of a crash (even if not used). Better have a defined, unused value, than accidentally using an uninitialized one.